frontend/src/e2e-test/specs/0_citizen/citizen-child-documents.spec.ts

```diff
@@ -75,13 +111,25 @@ beforeEach(async () => {
 })
 
 describe('Citizen child documents listing page', () => {
-  test('Published document is in the list', async () => {
+  test('Published hojks is in the list', async () => {
+    await page.reload()
```

Comment:
Why do both tests have `page.reload()`? I don't see a reason. All the test data has been set up before the page is opened.

In any case, it's better to avoid reloading the page because it takes some time. The usual reason is that we want to reflect some new data that was inserted after the page was initially loaded, but that can usually be worked around.

Reply:
uhm, maybe some copy-paste-mistake